src/main/java/org/radarcns/webapp/resource/DataSetEndPoint.java

```diff
@@ -155,20 +151,24 @@ public Dataset getSamplesJson(
             @Alphanumeric @PathParam(SOURCE_ID) String sourceId,
             @Alphanumeric @PathParam(SOURCE_DATA_NAME) String sourceDataName,
```

Comment:
Perhaps for a later stage: use Jersey parameter regex syntax like in MP, instead of using an annotation, e.g.
`/{subject: a-zA-Z0-9}/`